app/scenes/channel_info/channel_info.js

```diff
@@ -111,7 +158,7 @@ export default class ChannelInfo extends PureComponent {
                         <View style={[style.separator, {backgroundColor: this.props.theme.centerChannelBg}]}/>
                     </View>
                     <ChannelInfoRow
-                        action={() => true}
+                        action={() => this.handleLeave()}
```

Comment:
I added isRender here which basically just does not render the row if it is falsey.  We could always wrap the component if this is not desired.